bin/npm-cli-login.js

```diff
@@ -22,7 +22,7 @@ var login = function () {
     var scope = (found === -1) ? process.env.NPM_SCOPE : process.argv[found];
 
     found = getArg('--config-path', false);
-    var configPath = (found === -1) ? process.env.CONFIG_PATH : process.argv[found];
+    var configPath = (found === -1) ? process.env.NPM_CONFIG_PATH : process.argv[found];
```

Comment:
Using just CONFIG_PATH seemed too generic, it might conflict with other dependencies in projects. Can we use NPM_RC_PATH?